src/App.tsx

```diff
@@ -1,10 +1,55 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import { FC, useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer';
+import { TodoList } from './components/TodoList';
+import { Notification } from './components/Notification';
+import { SortTypes } from './types/SortTypes';
+import { Error } from './types/Error';
+
+const USER_ID = 10320;
+
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [activeFilter, setActiveFilter] = useState<SortTypes>(SortTypes.All);
+  const [errorMessage, setErrorMessage] = useState<Error | null>(null);
+
+  const handleError = () => {
+    setErrorMessage(Error.LOAD);
+    setTimeout(() => {
+      setErrorMessage(null);
+    }, 3000);
+  };
+
+  const loadTodos = async () => {
+    try {
+      const response = await getTodos(USER_ID);
+
+      setTodos(response);
+    } catch (error: unknown) {
+      handleError();
```

Comment:
Pass the error message to handleError() function to display the specific error occurred.

Reply:
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/49568391/163bfd4d-2302-4b51-93d6-fa0dd2539ad6)

Rebuild logic about this point